src/API/index.js

```diff
@@ -57,13 +57,17 @@ export const columnsAPI = {
     return axios.get(`${baseURL}api/board-columns`, { params: { boardId } })
   },
 
-  deleteColumnAPI(columnId) {
-    return axios.delete(`${baseURL}api/board-columns`, { params: { columnId } })
+  deleteColumnAPI(columnId, boardId) {
+    return axios.delete(`${baseURL}api/board-columns`, { params: { columnId, boardId } })
   },
 
   updateColumnAPI(columnId, title) {
     return axios.put(`${baseURL}api/board-columns`, { columnId, title })
-  }
+  },
+
+  columnDragAPI(dragId, dropId, boardId) {
+    return axios.put(`${baseURL}api/board-columns/dnd`, { dragId, dropId, boardId })
+  },
```

Comment:
Remove this `Drag, DnD...` from all names/variables (e.x you can use `change...`). And you don't need to wrap every function to objects, this instance connected with columns, so you can create only one object `columnAPI` and put all API calls inside of it